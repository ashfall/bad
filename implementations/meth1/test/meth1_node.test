#!/bin/sh

mkdir -p ${srcdir}/.test-tmp
rm -rf ${srcdir}/.test-tmp/out

${srcdir}/app/meth1_node 9000 \
  ${srcdir}/test/in.s0000.e1000.recs &
NODE_PID=$!

sleep 2

${srcdir}/app/meth1_client \
  500 ${srcdir}/.test-tmp/out first "127.0.0.1:9000" \
  | grep "^first," > .test-tmp/meth1_node.out

kill $NODE_PID

diff \
  ${srcdir}/test/meth1_node.out \
  ${srcdir}/.test-tmp/meth1_node.out

