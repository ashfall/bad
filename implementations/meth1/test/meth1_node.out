first, 00675bef0e528c74d1e4
cmd-first, 0, 1
